components/Card.module.css

```
@@ -16,206 +26,49 @@
-    border-bottom-left-radius: 20px;
-    border-bottom-right-radius: 20px;
-  }
-
-  .dateDiffContainer {
-    display: flex;
-    justify-content: space-between;
-    color: #666;
-
-    /* Linkbrary/caption */
-    font-size: 13px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: normal;
-  }
-
-  .description {
-    width: 300px;
-    height: 48px;
-    display: -webkit-box; /* 웹킷 브라우저에서 다중 줄 형태를 지원하기 위해 필요한 속성 */
-    -webkit-box-orient: vertical; /* 세로 방향으로 텍스트를 표시 */
-    overflow: hidden; /*텍스트가 div의 영역을 넘어갈 때 숨김 */
-    -webkit-line-clamp: 2; /* 두 줄을 넘어가면 생략 부호로 처리 */
-    font-size: 16px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 24px; /* 150% */
-  }
+.img:hover {
+  transform: scale(1.3);
+  border-radius: 20px;
 }
 
-@media (min-width: 1125px) {
-  .container {
-    position: relative;
-    width: 340px;
-    height: 334px;
-    border-radius: 20px;
-    display: flex;
-    flex-direction: column;
-    box-shadow: 0px 5px 40px 0px rgba(0, 0, 0, 0.16);
-  }
-
-  .img {
-    width: 340px;
-    height: 240px;
-    overflow: hidden;
-    image-rendering: crisp-edges;
-    border-top-left-radius: 20px;
-    border-top-right-radius: 20px;
-    object-position: center;
-    transition: transform 0.3s ease;
-  }
-
-  .img:hover {
-    transform: scale(1.3);
-    border-radius: 20px;
-  }
-
-  .textContainer {
-    background-color: white;
-    z-index: 1;
-    display: flex;
-    flex-direction: column;
-    gap: 10px;
-    padding: 15px 20px;
-    border-bottom-left-radius: 20px;
-    border-bottom-right-radius: 20px;
-  }
-
-  .dateDiffContainer {
-    display: flex;
-    justify-content: space-between;
-    color: #666;
+.textContainer {
+  background-color: white;
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 15px 20px;
+  border-bottom-left-radius: 20px;
+  border-bottom-right-radius: 20px;
+}
 
-    /* Linkbrary/caption */
-    font-size: 13px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: normal;
-  }
+.dateDiffContainer {
+  display: flex;
+  justify-content: space-between;
+  color: #666;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
 
-  .description {
-    width: 300px;
-    height: 48px;
-    display: -webkit-box; /* 웹킷 브라우저에서 다중 줄 형태를 지원하기 위해 필요한 속성 */
-    -webkit-box-orient: vertical; /* 세로 방향으로 텍스트를 표시 */
-    overflow: hidden; /*텍스트가 div의 영역을 넘어갈 때 숨김 */
-    -webkit-line-clamp: 2; /* 두 줄을 넘어가면 생략 부호로 처리 */
-    font-size: 16px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 24px; /* 150% */
-  }
+.description {
+  width: 300px;
+  height: 48px;
+  display: -webkit-box; /* 웹킷 브라우저에서 다중 줄 형태를 지원하기 위해 필요한 속성 */
+  -webkit-box-orient: vertical; /* 세로 방향으로 텍스트를 표시 */
+  overflow: hidden; /*텍스트가 div의 영역을 넘어갈 때 숨김 */
+  -webkit-line-clamp: 2; /* 두 줄을 넘어가면 생략 부호로 처리 */
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px; /* 150% */
```

픽셀이 아닌 퍼센트로 속성을 적용한다면 font-size가 변경되는 경우마다 line-height를 바꿔주지 않아도 된다는 이점이 있습니다!
```suggestion
  line-height: 150%;
```